### DISCUSSION TITLE
Does Shlink log ips?

### DISCUSSION THREAD

**Discussion by ghost** [shlinkio/shlink#1646] [Help wanted] [answered]
Does shlink log ips, i cant find this in the UI but does it still log them? If so, how can i disable it

**Comment by acelaya** [comment, marked as answer]
https://shlink.io/documentation/tracking-visits/#disable-tracking

**Reply by ghost** [reply]
Thank you. But by default if doesnt log raw ip addresses, only location data based on those ip addresses, right? Is there any setting for that or

**Reply by acelaya** [reply]
By default Shlink persists an anonymized version of the IP address https://shlink.io/documentation/tracking-visits/#considerations

You can disable this under your own risk and make it persist the actual IP address.

**Reply by ghost** [reply]
Thank you for the help and the amazing project:)
